mlflow/server/auth/__init__.py

```diff
@@ -119,25 +442,129 @@ def create_user():
     if content_type == "application/x-www-form-urlencoded":
         username = request.form["username"]
         password = request.form["password"]
+
+        if store.has_user(username):
+            flash(f"Username has already been taken: '{username}'", category="error")
+            return redirect(ROUTES.SIGNUP)
+
+        store.create_user(username, password)
+        flash(f"Successfully signed up user: '{username}'")
```

Comment:
@gabrielfu Does the message flashing work fine on your machine?

Reply:
No it's not showing either. I think it's because "/" is not handling flashed messages right now. Should I create something like an alert window to show the flashed messages?